subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/DependencyInsightReportTaskIntegrationTest.groovy

```diff
@@ -493,10 +491,12 @@ org:foo:1.+ FAILED
 
         then:
         outputContains """
-org:leaf:1.0 (forced)
+org:leaf:1.0
    variant "runtime" [
       org.gradle.status = release (not requested)
    ]
+   Selection reasons:
+      - Forced : module org:leaf:1.0
```

Comment:
What's the benefit of repeating the module and version here?

Reply:
Actually I'm not certain this is worth it, but there are 2 reasons:
1) We differentiate between a forced _module_ (`resolutionStrategy.force()`) and a forced _dependency_ (`force = true`)
2) There are (rare) cases where the version chosen is not the forced version. This can happen with dependency substitution rules. I'll see if I can add a test case.

Reply:
Actually, in the second case we don't (currently) show the `force` reason at all. So I'm not sure this change is worth it.

Reply:
I've removed this from the PR. We can add it back in later if we decide it's better.